docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,92 @@
+
+During a round, processes can exchange different messages and we can summarize the execution in two voting steps: Prevote and Precommit steps. A vote can be for a particular value or for a null value.
+
+A correct process decides on a value *v* upon receiving a proposal and *2f + 1* Precommit messages for *v* in a round *r*. A correct process sends a Precommit message for a value *v* upon receiving a proposal and *2f + 1* Prevote messages for *v* in a round *r*.
+
+Validators wait some time before sending a Prevote with a null value if they don't receive a valid proposal after a certain time and they send a Precommit message with a null value if they don't receive *2f + 1* Prevote messages for a value.
+If a correct process receives more than 2/3 Precommit messages for a null value in the same round, it moves to the next round.
+
+After a decision has been made, processes continue to agree on other values on another height (consensus instance) and they repeat the process described above in order to agree on different transactions.
+
+In order to ensure that processes will eventually come to a decision, there are some constraints that are applied. These rules are used to prevent any malicious attempt to cause more than one block to be committed at a given height. 
+When a validator sends a Precommit for a value *v* at round *r*, we say it is *locked* on *v*. Validators can propose and send Prevote messages for a value they have locked and they can only change the locked value if they receive a more recent proposal with a quorum of Prevote messages. 
+
+These conditions ensure both the safety and liveness properties of the consensus algorithm because a validator cannot send a Precommit message without sufficient evidence and cannot send a Precommit message for a different value at the same time.
+
+Tendermint is notable for its simplicity and performance and is currently one of the best consensus algorithm used in production.
+
+## Properties of consensus 
+
+The Tendermint consensus algorithm guarantees the following properties:
+
+- **Agreement**: No two correct processes decide on different values
+
+- **Validity**: A value decided is a valid value, respect to the specifications of validity of a message
+
+- **Termination**: All correct processes eventually decide
+
+- **Integrity**: No correct process decides more than once in a single height 
+
+We limit the discussion to the agreement property only because its proof can be useful to better understand the fork accountability problem. 
+We invite interested readers to read the official paper for a more formal, complete and rigorous proof of all the properties listed above.
+
+## Agreement property
+
+In this section we'll give a simple proof of the agreement property for the Tendermint algorithm and show why the majority assumption is important for ensuring this property.
```

Comment:
I would avoid using contractions in order to look more formal.

Reply:
What is majority assumption? Be more precise here because someone may not know what you mean here.

Reply:
I think I removed all the contractions from all the documentation files. Majority assumption better explained.